contentcuration/contentcuration/frontend/channelList/views/ChannelListIndex.vue

```diff
@@ -139,7 +136,7 @@
         return this.lists.includes(this.$route.params.listType);
       },
       homeLink() {
-        return window.Urls.base();
+        return this.libraryMode ? window.Urls.base() : window.Urls.channels();
```

Comment:
Would it make sense to add a `home()` url?

Reply:
That makes sense, although for now we'll need to leave it as is in order to support the catalog workflow where the base `/` url opens the catalog. If we leave it for the regular Studio workflow though, it redirects to the login page if the user isn't signed in